devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy

```diff
@@ -225,6 +234,19 @@ def refresh() {
 	delayBetween(commands,100)
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	runEvery5Minutes(healthPoll)
+	log.debug "Configuring Reporting and Bindings."
+	// Device-Watch allows 2 check-in misses from device
+	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
The hub-based polling system sends switch events every 15 minutes in the absence of state changes to let the cloud know that the device is still being polled. The checkInterval should be at least 16 minutes.

The hub also sends an offline alert approx. 5 minutes after a polled switch stops responding, so we could have better responsiveness by making use of those events to mark the device offline.